modules/tile-converter/src/slpk-extractor/helpers/file-handle-file.ts

```diff
@@ -1,17 +1,17 @@
-import {FileProvider} from '@loaders.gl/i3s';
+import {FileProvider} from '@loaders.gl/zip';
 import {FileHandle} from './fs-promises';
 
 /**
  * Provides file data using node fs library
  */
-export class FileHandleProvider implements FileProvider {
+export class FileHandleFile implements FileProvider {
   /**
-   * Returns a new copy of FileHandleProvider
+   * Returns a new copy of FileHandleFile
    * @param path The path to the file in file system
    */
-  static async from(path: string): Promise<FileHandleProvider> {
+  static async from(path: string): Promise<FileHandleFile> {
```

Comment:
Nit:
- `.from` seems overly generic. Should we call this `.openFile` or something more descriptive?
- Since all methods are async anyway you don't really need this static async constructor. You could just create FileHandleFile objects with the normal constructor and  just begin by calling `await this._openFile()` in all methods to make sure the file was open before you read.